src/localization/en.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
As the app grows we will want to structure this more, for example could logout and currentLocale strings be in a header object?